crates/metadata-server/src/raft/store.rs

```diff
@@ -540,11 +544,28 @@ impl Member {
         // todo remove additional indirection from Arc
         connection_manager.store(Some(Arc::new(new_connection_manager)));
 
-        // todo properly configure Raft instance
+        let_assert!(
+            MetadataServerKind::Raft(raft_options) = &Configuration::pinned().metadata_server.kind,
+            "Expecting that the embedded/raft metadata server has been configured"
+        );
+
         let mut config = Config {
             id: to_raft_id(my_member_id.node_id),
+            election_tick: raft_options.raft_election_tick,
+            heartbeat_tick: raft_options.raft_heartbeat_tick,
             read_only_option: ReadOnlyOption::Safe,
-            ..Default::default()
+            check_quorum: true,
+            pre_vote: true,
+            // 64 KB
```

Comment:
super nit picky. KB = 1000B, KiB = 1024B. I know that we don't use that consistently across the code base, but we probably should

Reply:
Yes, this should be correct.